util/fileutil/homedirs.go

```diff
@@ -0,0 +1,67 @@
+package fileutil
+
+import (
+	"bufio"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/user"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// GetUserHomeDirs returns a map of all users and their homedirs
+func GetUserHomeDirs() (map[string]string, error) {
```

Comment:
I feel that this is too much logic for what is normally needed... but hey... I guess it's now already written.

Reply:
this code was moved here from our `autocompletion.getUsers()`, no new code was written in this PR for this function.